tests/app/app_routes/admin/test_admins_required.py

```
@@ -0,0 +1,77 @@
+"""
+Tests for admin_required decorator.
+"""
+from unittest.mock import MagicMock, patch
```

Flake8 (7.3.0)

[error] 'unittest.mock.MagicMock' imported but unused

(F401)